xtable-core/src/main/java/org/apache/xtable/iceberg/IcebergTableManager.java

```diff
@@ -100,6 +101,14 @@ Table getOrCreateTable(
     }
   }
 
+  Table loadHadoopTables(HadoopTables hadoopTables, String basePath, String tableName) {
```

Comment:
Instead of manipulating the name at this level, it may be easier to just update the IcebergConversionSource which produces the `InternalTable` object

Reply:
Good point. Done.

Reply:
I was referring to this section of the code: https://github.com/apache/incubator-xtable/pull/630/files#diff-aa668462e6a6dcd3a7ebee90de2678602f7cc68448fcf438a64ed593c221f39fR113

Reply:
Fixed